src/Interop/CSharpDafnyASTInterop.dfy

```diff
@@ -19,4 +22,11 @@ module {:extern "CSharpDafnyASTInterop"} Bootstrap.Interop.CSharpDafnyASTInterop
       : (ty': CSharpDafnyASTModel.Type)
       ensures TypeHeight(ty') <= TypeHeight(ty)
   }
+
+  class {:extern} ExprUtils {
+    constructor {:extern} () requires false // Prevent instantiation
+
+    static function method {:extern} UnescapedCharacters(ty: CSharpDafnyASTModel.CharLiteralExpr)
+      : (cs: IEnumerable<char>)
```

Comment:
I'm not super comfortable exposing `IEnumerable`s, since they are fundamentally mutable things.
Also, do we need a `reads` clause here?

Reply:
It's now a `List` and has a `reads` clause.